community/datadog/templates/agent-clusterchecks-deployment.yaml

```diff
@@ -0,0 +1,110 @@
+{{- if and .Values.clusterAgent.enabled .Values.datadog.clusterChecks.enabled .Values.clusterChecksRunner.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "datadog.fullname" . }}-clusterchecks
+  labels:
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    app.kubernetes.io/name: "{{ template "datadog.fullname" . }}"
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
```

Comment:
We expect to see [our metering annotations in deployments.](https://github.com/IBM/charts/blob/master/deprecated/GUIDELINES.md#metering-integration)

Reply:
Github